### ISSUE TITLE
render with more than 3 view

### ISSUE THREAD

**Issue by ds22058** [donydchen/matchnerf#2]
Hi, I try your demo and the result is really great. It seems that the matchnerf_3v_ibr.pth can only render scene with 3 views. What should I do to render a scene with more than 3 views such as orchid in LLFF dataset(25 views)? Also, I wanna know how to adjust the render poses. Thanks in advance!

**Comment by donydchen** [comment]
Hi @ds22058 , We sincerely appreciate your interest in and acknowledgement of our efforts.

Since the released model was trained using only the 3 nearest views as input, it cannot use any other number of input views *directly*. Nonetheless, for the scenario with a larger number of input views, it is still possible to use the pretrained model by employing an engineering strategy such as sliding windows. Specifically, for each given novel viewpoint, locate the 3 nearest viewpoints from the source views (25 views) and use those 3 views as input to render the corresponding novel view. 

Regarding render poses, as MatchNeRF relies on a *relative coordinate system* to generalize better to unseen scenes, there is no need to adjust camera poses to match with the training camera distribution. See related code snippets below, 

https://github.com/donydchen/matchnerf/blob/08c0943454c57c84f60083722fd1e6594528bc0e/datasets/colmap.py#L85-L95 

In other words, our pretrained model should be able to render reasonable outputs from your own data using COLMAP extracted camera poses. For instance, as detailed at https://github.com/donydchen/matchnerf#use-your-own-data, the camera poses of our demo are directly extracted by running [img2poses.py](https://github.com/Fyusion/LLFF/blob/master/imgs2poses.py).

Feel free to let us know if additional details are required. Thanks.
